### DISCUSSION TITLE
--reload-include does not work on .env file

### DISCUSSION THREAD

**Discussion by imlison** [Kludex/uvicorn#1705] [Potential Issue] [answered]
I want to reload after .env file changed, but --reload-include=.env not work. it seems be ignored by --reload-exclude default settings. however, using --reload-include=.*  to overwrite these defaults will watching all hidden files  not just .env. What is the right way?

**Comment by Kludex** [comment, marked as answer]
Let's continue on https://github.com/encode/uvicorn/issues/2166.

**Comment by adithya04dev** [comment]
its still not reloading after the .env file changes
